tests/client/amo/components/TestSearchResult.js

```diff
@@ -4,15 +4,21 @@ import {
   findRenderedDOMComponentWithClass,
   renderIntoDocument,
 } from 'react-addons-test-utils';
+import { Provider } from 'react-redux';
 
+import createStore from 'amo/store';
 import SearchResult from 'amo/components/SearchResult';
 import { getFakeI18nInst } from 'tests/client/helpers';
 
 
 describe('<SearchResult />', () => {
   function renderResult(result) {
+    const initialState = { api: { clientApp: 'android', lang: 'en-GB' } };
+
     return findRenderedComponentWithType(renderIntoDocument(
-      <SearchResult i18n={getFakeI18nInst()} result={result} lang="en-US" />
+      <Provider store={createStore(initialState)}>
```

Comment:
Too bad we need to wrap all this stuff with `<Provider>` now. I guess there's always a trade off.

It might be kind of nice to simplify this though (not here just thinking out loud):

```jsx
function renderWithProvider(node, { store = createStore() }) {
  const name = node.type;  // I'm not sure if this works?
  return findRenderedComponentWithType(renderIntoDocument((
    <Provider store={store}>{node}</Provider>
  ), name)).getWrappedInstance();
}